app/views/devices/index.html.erb

```diff
@@ -0,0 +1,84 @@
+<%= render 'shared/partials/nav-global.html' %>
+<main class="devices">
+  <%= render 'shared/partials/header-global.html' %>
+  <nav class="device__selection">
+    <ul>
+      <li class="device__keurig device-selected">
+        <a href="/device-keurig">
+          <%= image_tag("illustrations/ill-keurig.svg", :alt => "Keurig") %>
+          <h3>Keurig</h3>
+        </a>
+      </li>
+
+      <li class="device__coffee-maker">
+        <a href="/device-coffee-maker">
+          <%= image_tag("illustrations/ill-coffee-maker.svg", :alt => "Coffee Maker") %>
+          <h3>Coffee Maker</h3>
+        </a>
+      </li>
+
+      <li class="device__french-roast">
+        <a href="/device-french-roast">
+          <%= image_tag("illustrations/ill-french-roast.svg", :alt => "French Roast") %>
+          <h3>French Roast</h3>
+        </a>
+      </li>
+    </ul>
+  </nav>
+
+  <section class="device__problems">
+    <ul>
+      <li>
+        <a href="">
+          <h4>Lukewarm coffee chills my soul</h4>
+          <%= image_tag("icon-caret-hover.svg") %>
```

Comment:
I suggest setting `<% hover-icon = image_tag("illustrations/icon-caret-hover.svg", :alt => " ") %>` close to the head of the page and then doing a `<%= hover-icon %>` at each of these declarations for the sake of file size.